bulk-export/src/test/java/no/sikt/nva/data/report/api/export/CsvTransformerTest.java

```diff
@@ -89,7 +90,7 @@ void shouldWriteCsvFileToS3ForAllReportTypes(ReportType reportType) throws IOExc
     }
 
     @Test
-    void shouldWriteCsvFilesForAlleReportTypesToSpecificFolderInExportBucket() throws IOException {
+    void shouldWriteCsvFilesForAlleReportTypesToSpecificFolderInExportBucketWithCsvFileExtension() throws IOException {
```

Comment:
Could this be shortened to something easier to read? For example, "shouldHaveFileExtension", "shouldAddFileExtension", or something else?

Alternatively, would it make sense to split the test in two if it is intended to both test that files are written and that the file extension is correct?

Reply:
Agreed, splitting into it's own test 👍